data/merchant intro.txt

```diff
@@ -0,0 +1,138 @@
+				`	"Of course."`
+					accept
+				`	"No thanks."`
+					decline
+	on complete
+		payment 30000
+		conversation
+			`Men quickly unload the cargo from your hold. A woman in a dark blue suit, which is almost the same suit as the man you met for this job, thanks you. "We have another mission that requires a captain to transport it. If you would like to take that offer, meet me in the spaceport after this."`
+
+mission "Merchant Start 2"
+	name "Passenger transport"
+	description "Help out the merchants by bringing a family to <destination> by <date>"
+	passengers 3
+	deadline 7
+	to offer
+		has "Merchant Start 1: done"
+	source
+		government "Republic" "Syndicate" "Free Worlds" "Neutral"
+	destination
+		distance 5 5
+		government "Republic" "Free Worlds" "Syndicate" "Neutral"
+	on offer
+		conversation
+			`You walk into the spaceport, looking for the lady. You quickly find her striking dark blue suit in the crowd, clearly standing out from everyone's frivolous apparel as you walk over to her. "What's the job you're offering me?" She smiles, "Well, this family desperately needs transport to <destination> by <date>," she says, pointing out a group of people at a corner of the spaceport, "they're in a bit of troublesome economic crisis at the moment"`
+			choice
+				`	"I'll do it."`
+					accept
+				`	"I don't have the time."`
+					decline
+	on complete
+		payment 40000
+		conversation
+			`The family thanks you profusely, grateful for meeting such a generous captain. You watch the family walk off and exchange greetings with another man in a blue suit, the blue-suited man walks up to you and says, "Meet me in the spaceport," he instructs. "Don't dally."`
+
+mission "Merchant Start 3"
+	name "Special transport"
+	description "Get <bunks> travelers with <cargo> to <destination>"
+	passengers 2
+	cargo "traveling goods" 10
+	to offer
+		has "Merchant Start 2: done"
+	source
+		government "Republic" "Syndicate" "Free Worlds" "Neutral"
+	destination "New Switzerland"
+	npc
+		government "Pirate"
+		personality entering plunders
+		fleet "Bounty Hunters"
+	on offer
+		conversation
+			`"Captain," salutes the man. "This job involves <bunks> travelers trying to get to <destination> with <cargo>. Now, I have no idea why they need so much of it, but get it over there alright?"`
+			choice	
+				`	"Yes, sir."`
+					accept
+				`	"Sorry, but I can't."`
+					decline
+	on complete
+		payment 50000
+		conversation
+			`You help the travelers lug their cargo away, still wondering why so much of it is needed. The first merchant you met walking up to you. "Well, Captain, may I ask you for your name?"`
+			choice
+				`	"It's <first> <last>."`
+					goto name
+				`	"I would rather not say."`
+					goto anonymous
+			label name
+			`	"Thank you, <first>."
+				goto next
+			label anonymous
+			`	The man smiles slightly. "That's perfectly fine, Captain <last>," he says, surprising you. "Just do a better job of covering your pilot's license next time."`
```

Comment:
Since this guy is a recurring character now, how about he gives his name? From here on it will be easier to know who he is in future missions.

Reply:
@Fzzr The three characters would be mentioned in... eh, either the next mission or the mission after that. I didn't keep track of the total mission count that tech made atm, but they'll be introduced.
On a side note, is it just me, or can you see someone's name when you're pinging them, or is that a username issue?

Reply:
@TheMarksman-ES Yes, I can see someone's name when I ping them. No idea what might break that. No strong feelings about naming them immediately, just as long as they don't remain anonymous indefinitely.

Reply:
I intend to name him, well he's named three missions from then. I guess that I should do it earlier since I'm intending to add a few more missions between that one and this one